beacon-chain/sync/initial-sync/initial_sync_test.go

```diff
@@ -37,17 +37,17 @@ import (
 
 type testCache struct {
 	sync.RWMutex
-	rootCache       map[uint64][32]byte
-	parentSlotCache map[uint64]uint64
+	rootCache       map[types.Slot][32]byte
+	parentSlotCache map[types.Slot]types.Slot
 }
 
 var cache = &testCache{}
 
 type peerData struct {
-	blocks         []uint64 // slots that peer has blocks
+	blocks         []types.Slot // slots that peer has blocks
```

Comment:
`blocks` is a []slot?

Reply:
yes, it is a misnomer on data structure type -- but typed correctly.